apps/spark/src/main/java/com/linkedin/openhouse/jobs/scheduler/tasks/OperationTasksBuilder.java

```diff
@@ -19,6 +27,11 @@
 @Slf4j
 @AllArgsConstructor
 public class OperationTasksBuilder {
+  private static final double COMPUTE_COST_WEIGHT_DEFAULT = 0.5;
+  private static final double COMPACTION_GAIN_WEIGHT_DEFAULT = 0.5;
```

Comment:
```suggestion
  private static final double COMPUTE_COST_WEIGHT_DEFAULT = 0.3;
  private static final double COMPACTION_GAIN_WEIGHT_DEFAULT = 0.7;
```

I have been biasing towards compaction gain in my analysis. Let's start with this and then tune later.